app/lib/bk/compat/models/steps/command.rb

```diff
@@ -1,111 +1,9 @@
 # frozen_string_literal: true
 
+require_relative 'wait'
+
```

Comment:
You are missing other classes used here:

```suggestion
require_relative 'base'
require_relative 'block'
require_relative 'wait'
require_relative '../plugin'
```

Reply:
Adjusted 👍